recipe/gen_patch_json.py

```diff
@@ -1232,6 +1232,25 @@ def _gen_new_index(repodata, subdir):
             elif record["version"] == "0.7.14":
                 _replace_pin("python >=2.7", "python >=2.7,<3.10", deps, record)
 
+        if record_name == "conda-build":
+            # Work around vendored auxlib import error; see:
+            #   https://github.com/conda/conda-build/issues/4333
+            #   https://github.com/conda/conda-build/pull/4335
+            #   https://github.com/conda-forge/conda-build-feedstock/pull/169
+            #
+            # Conda 4.11.1 should fix this upstream, so all we really need to
+            # do is avoid 4.11.0 specifically.
+            major, minor, patch = (int(v) for v in record["version"].split("."))
+            build_num = record['build_number']
+            if (major < 3 or (major == 3 and minor < 21) or
+                (major == 3 and minor == 21 and patch < 6) or
+                (major == 3 and minor == 21 and patch == 6 and build_num < 2)
+                ):
+                if "constrains" in record:
+                    record["constrains"].append("conda !=4.11.0")
+                else:
+                    record["constrains"] = ["conda !=4.11.0"]
```

Comment:
Probably not what you'd want, c.f. https://github.com/conda-forge/conda-feedstock/pull/108#issuecomment-562680818
(but I'm also not sure what the actual behavior will be).

Reply:
@mbargull had to reread the 4.4.0 release log a few times...so since `conda-build` actually depends on `conda` this isn't a `constrains` (which is optional) instead we want to inject this requirement as a `depends`?

```suggestion
                for i, d in enumerate(record["depends"]):
                    if dep in d:
                        record["depends"][i] += ",<4.11.0"
                        break
                else:
                    record["depends"].append("conda <4.11.0")
```